features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/operationUtils.js

```diff
@@ -200,6 +200,24 @@ function isSelectAll(selectedOperations, operations) {
     }
     return true;
 }
+
+/**
+ * Check whether the provided object is a Ref object.
+ *
+ * @param {object} content : The object that needs to be checked.
+ * @return {boolean} true if Ref, false otherwise.
+ * */
+function isRef(content) {
+    let isReference = false;
+    if (typeof content === 'object') {
+        Object.keys(content).map((name) => {
+            isReference = name === '$ref';
```

Comment:
it is not good to update block scope local variable from a lambda function, why do we need to loop through the object keys ?

if it's an object then for sure it's not a $ref ?